### PR TITLE
some text added and cleaned up bullets

### PR DESCRIPTION
**Pull request by taleena**

### CODE REVIEW

File: noises.md [hunk 1]
```diff
@@ -194,3 +238,4 @@ https://www.youtube.com/watch?v=31zzMb3U0iY
 
 ### Loogie
 - Start by making the Hard H sound, then pull your lips back quickly
+custom glue-up
```
Comment (nickvisut) [inline noises.md]:
```suggestion
```

Comment (nickvisut) [reply noises.md]: Screen Shot 2019-03-19 at 1 37 07 AM

weird perms issues so can't fix this nit thru GitHub UI

Comment (nickvisut) [reply noises.md]: maybe b/c `master` is protected /shrug

**Comment by taleena** [comment]
That’s weird, I’ll check my original file for that, no idea where it came from!

**Pull request merged**